vision/setup.py

```diff
@@ -20,21 +21,38 @@
 
 PACKAGE_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(PACKAGE_ROOT, 'README.rst')) as file_obj:
-    README = file_obj.read()
+with io.open(os.path.join(PACKAGE_ROOT, 'README.rst'), 'r') as readme_file:
+    readme = readme_file.read()
 
-# NOTE: This is duplicated throughout and we should try to
-#       consolidate.
-SETUP_BASE = {
-    'author': 'Google Cloud Platform',
-    'author_email': 'jjg+google-cloud-python@google.com',
-    'scripts': [],
-    'url': 'https://github.com/GoogleCloudPlatform/google-cloud-python',
-    'license': 'Apache 2.0',
-    'platforms': 'Posix; MacOS X; Windows',
-    'include_package_data': True,
-    'zip_safe': False,
-    'classifiers': [
+cur_dir = os.path.realpath(os.path.dirname(__file__))
+with io.open('%s/requirements.txt' % cur_dir) as requirements_file:
```

Comment:
Let's discuss soon. I actually prefer this form, but you might have reasons I have not considered.